app-emulation/fuse/fuse-1.5.2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Free Unix Spectrum Emulator by Philip Kendall"
+HOMEPAGE="http://fuse-emulator.sourceforge.net"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="alsa ao backend-fbcon backend-sdl backend-svga backend-X gpm joystick memlimit png xml"
+
+# Only one UI back-end can be enabled at a time
+REQUIRED_USE="?? ( backend-fbcon backend-sdl backend-svga backend-X )"
+
+RDEPEND=">=app-emulation/libspectrum-1.4.1
+	alsa? ( media-libs/alsa-lib )
+	ao? ( media-libs/libao )
+	backend-sdl? ( media-libs/libsdl )
+	backend-svga? ( media-libs/svgalib )
+	backend-X? ( x11-libs/libX11 x11-libs/libXext )
+	!backend-fbcon? ( !backend-sdl? ( !backend-svga? ( !backend-X? ( x11-libs/gtk+:3 ) ) ) )
+	dev-libs/glib:2
```

Comment:
You almost got the sorting right except this. unconditional deps always preceding use-conditional deps. But I'm fixing that for you.